### PR TITLE
Added unit/accept/cover to jenkinsfile

### PR DESCRIPTION
**Pull request by AdamSaleh**

### CODE REVIEW

File: fh-messaging/lib/fh_rollup.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ exports.process = function(config, logger, inputDate, daysToKeep, cb) {
   logger.info(logMessage);
 
   var msgProcess;
+
```
Comment (fh-qube) [inline fh-messaging/lib/fh_rollup.js]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: Jenkinsfile [hunk 2]
```diff
@@ -10,17 +10,69 @@ fhBuildNode {
...
+                sh "cp -R config /tmp/fh-messaging-config"
+                sh """
+                  cat /tmp/fh-messaging-config/dev.json | \
+                  jq '.database.host="${env.MONGODB_HOST}"' | \
+                  jq '.metrics.database.host="${env.MONGODB_HOST}"' | \
+                  jq '.metrics.metricsDir="/tmp/var/log/feedhenry/fh-messaging/metrics"' | \
+                  jq '.configDir="/tmp/fh-messaging-config/"' > \
+                  /tmp/fh-messaging-config/conf.json
+                """
+                sh "cp /tmp/fh-messaging-config/conf.json config/dev.json"
```
Comment (mikenairn) [inline Jenkinsfile]: @AdamSaleh Were we not able to resolve this in the tests themselves like we have for other components?

File: fh-metrics/test/test_helper.js [hunk 3]
```diff
@@ -23,6 +23,10 @@ nconf.set("metrics:ssl:cert", path.resolve(__dirname, '../config/server.crt'));
 nconf.set("metrics:ignoreAPIKey", true);
 
 exports.getConfig = function () {
+  if(process.env.MONGODB_HOST) {
+    console.log(process.env.MONGODB_HOST);
+    nconf.set("metrics:database:host",process.env.MONGODB_HOST);
+  }
```
Comment (mikenairn) [inline fh-metrics/test/test_helper.js]: Does this not get round the need to do the config changes in the Jenkinsfile?

Comment (AdamSaleh) [inline fh-metrics/test/test_helper.js]: Yes, for fh-metrics. Problem was with fh-messaging.

File: Jenkinsfile (hunk 2 above)
Comment (AdamSaleh) [inline Jenkinsfile]: There were too many places where the tests for fh-massaging read the file directly. This was the simpler, more robust solution.

**Comment by mikenairn** [comment]
If it works, lets go with it for now. Needs a rebase and then looks good to merge.

**Pull request merged**